TemplePlus/action_sequence.cpp

```diff
@@ -1227,7 +1227,10 @@ uint32_t ActionSequenceSystem::MoveSequenceParse(D20Actn* d20aIn, ActnSeq* actSe
 						}
 					} 
 					else {
-						if (!tbStatCopy.hourglassState || 	tbStatCopy.hourglassState == 4 && (config.preferUse5FootStep || !objects.IsPlayerControlled(d20a->d20APerformer))) { // added for AI to take 5' steps when it still has full round action to exploit
+						// Added for AI to take 5' steps when it still has full round action to exploit.  Players can select an option for them as well.  
+						// Don't use the player option for stunning fist since there wouldn't be a way to use it without the 5 foot step and full attack.
+						if (!tbStatCopy.hourglassState || 	tbStatCopy.hourglassState == 4 && ((config.preferUse5FootStep && (d20->globD20Action->d20ActType != D20A_STUNNING_FIST)) 
```

Comment:
I prefer a generic fix rather than ad-hocing specific actions (unless Stunning Fist is a real exception).

Reply:
For some reason stunning fist works differently than everything else according to issue #585.  The best way to fix the issue might be to rework the stunning fist radial menu.  Not sure we want to change that around though.

Reply:
Also it doesn't seem logical that the place where it should be fixed is inside MoveSequenceParse.
I'll look into this.

Reply:
It might be better to do this further up stream.  Not sure on that.  Other than that I think the pull request is ready.

Reply:
After reading the comments on the issue, should I revert this change and merge whats in there?

Reply:
Yes please.